app/people/knight.py

```diff
@@ -0,0 +1,40 @@
+class Knight:
+
+    def __init__(self, knight_info: dict) -> None:
+        self.name = knight_info["name"]
+        self.power = knight_info["power"]
+        self.hp = knight_info["hp"]
+        self.armour = knight_info["armour"]
+        self.weapon = knight_info["weapon"]
+        self.potion = knight_info["potion"]
+        self.protection = 0
+
+    def apply_armour(self) -> None:
+        if self.armour is not []:

```

Comment:
The condition `if self.armour is not []:` is not the correct way to check for an empty list. Use `if self.armour:` instead, which is more Pythonic and checks if the list is not empty.